src/routes/_components/status/Media.html

```diff
@@ -1,4 +1,4 @@
-{#if (type === 'video' && !blurhash) || type === 'audio'}
+{#if type === 'video' || type === 'audio'}
```

Comment:
you probably still want to have the case with or without blurhash different here. You do not want to have the play triangle to show in blurhash mode (I guess).

Reply:
Ah, I noticed that, but didn't think it was a big deal. You're right though, that triangle is weird-looking.

Reply:
OK, I fixed it. BTW it turns out that audio can be marked as sensitive too, although it's kind of weird to me, but I decided to treat it the same as video.